miniaudio.py

```diff
@@ -1268,11 +1268,16 @@ def on_close() -> None:
 
 def stream_with_callbacks(sample_stream: PlaybackCallbackGeneratorType,
                           progress_callback: Union[Callable[[int], None], None] = None,
+                          frame_process_method: Union[Callable[[array.array], None], None] = None,
```

Comment:
I've just added a docstring to this fuction in the master branch. 
Can you please sync your branch and add a short mention of the new parameter to the docstring